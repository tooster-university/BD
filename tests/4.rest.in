{"open": {"database": "student", "login": "app", "password": "qwerty"}}
{"actions": {"timestamp": 43451926, "member": 8, "password": "M70EHZ"}}
{"projects": {"timestamp": 98057069, "member": 6, "password": "3LHCQALDPM"}}
{"actions": {"timestamp": 122013180, "member": 7, "password": "Y2TNXWMFVX"}}
{"votes": {"timestamp": 161760913, "member": 6, "password": "3LHCQALDPM"}}
{"protest": {"timestamp": 169530745, "member": 7, "password": "Y2TNXWMFVX", "action": 103, "project": 304, "authority": 451}}
{"projects": {"timestamp": 188450982, "member": 6, "password": "3LHCQALDPM"}}
{"votes": {"timestamp": 213754999, "member": 4, "password": "NB4T7BZ9T"}}
