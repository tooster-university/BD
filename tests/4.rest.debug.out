{"status": "OK"}
{"status": "ERROR", "debug": "ERROR:  Member must be leader but doesnt exist"}
{"status": "ERROR", "debug": "ERROR:  Incorrect member password"}
{"status": "ERROR", "debug": "ERROR:  Member must be leader but doesnt exist"}
{"status": "ERROR", "debug": "ERROR:  Incorrect member password"}
{"status": "OK"}
{"status": "ERROR", "debug": "ERROR:  Incorrect member password"}
{"status": "ERROR", "debug": "ERROR:  Member inactive"}
